agent/agent-tooling/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/pipeline/TelemetryPipelineListener.java

```diff
@@ -73,5 +86,10 @@ public void onResponse(TelemetryPipelineRequest request, TelemetryPipelineRespon
     @Override
     public void onException(
         TelemetryPipelineRequest request, String errorMessage, Throwable throwable) {}
+
+    @Override
+    public CompletableResultCode shutdown() {
+      return CompletableResultCode.ofSuccess();
```

Comment:
same here.. it's doing nothing.. can we make the interface shutdown optional? not required for all listener where it return success and does nothing?

Reply:
let's reconsider what to do with these shutdowns once we think more about non-javaagent usage, they aren't really relevant for javaagent since we it lives for the length of the JVM process